docs/en/latest/config.json

```diff
@@ -162,6 +162,13 @@
             "discovery/eureka"
           ]
         },
+        {
+          "type": "category",
+          "label": "Examples",
```

Comment:
This section is not part of the category.

Reply:
then Shall I add this directly to the sidebar?I don't know how to add  https://github.com/apache/apisix/blob/master/docs/en/latest/examples/plugins-hmac-auth-generate-signature.md tor.Sorry
![10](https://user-images.githubusercontent.com/53715187/116961099-060ac700-acc0-11eb-8d64-80ddef6c6fd5.JPG)
![11](https://user-images.githubusercontent.com/53715187/116961101-07d48a80-acc0-11eb-8ea4-413a9d54f6a1.JPG)

Reply:
Don't add it to the sidebar. It doesn't belong to that.

Please revert the change.
Please explain yourself clearly when submitting the change.

I don't even know why you want to add it to the sidebar.

Reply:
So when visitors see this page https://apisix.apache.org/docs/apisix/examples/plugins-hmac-auth-generate-signature, how could they see other menus?

Reply:
Not all pages need a sidebar. They can go back from the browser.

Reply:
And this problem should be fixed by the doc render framework itself. Adding unrelative doc to the sidebar to fix it is **a hack**.